src/Form/Type/TodoStatusFilterType.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+
+namespace App\Form\Type;
+
+
+use App\Model\SearchData;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\ChoiceList\Loader\CallbackChoiceLoader;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Form\Extension\Core\Type as Types;
+
+class TodoStatusFilterType extends AbstractType
+{
+
+    private $translator;
+
+    /**
+     * TodoStatusFilterType constructor.
+     * @param TranslatorInterface $translator
+     */
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        /** @var SearchData $searchData */
+        $searchData = $builder->getData();
+
+        $translationDomain = 'search';
+
+        $builder->add('selectedStatus', Types\ChoiceType::class, [
+        'attr' => [
+            'onchange' => 'this.form.submit()',
+        ],
+        'choice_loader' => new CallbackChoiceLoader(function() use ($searchData) {
+            $translatedTitleAny = $this->translator->trans('any', [], 'form');
+            return array_merge([$translatedTitleAny => 0], $this->buildRubricsChoices($searchData->getStatuses()));
```

Comment:
Gut wäre, einen eigenständigen Methoden-Namen (und nicht `buildRubricsChoices()`) zu verwenden:
```suggestion
            return array_merge([$translatedTitleAny => 0], $this->buildTodoStatusChoices($searchData->getStatuses()));
```